src/components/material/material-buttons/online/MaterialButtonsOnline.tsx

```diff
@@ -49,7 +50,7 @@ const MaterialButtonsOnline: FC<MaterialButtonsOnlineProps> = ({
 
     //  We have experienced that externalUrl is not always valid.
     if (!isUrlValid(externalUrl)) {
-      return null;
+      throw new InvalidUrlError("The external url is not valid.");
```

Comment:
Please include the url in the error if possible.

Otherwise it can be hard to debug errors. If the message is intented for internal use then it could just be part of the error message. You could also add it as the context on the `IntervalidUrlError` class.

Reply:
We render the broken URL to the user. (Although I didn't add it as context for the class.